apps/epicdev-ai/src/app/(content)/workshops/[module]/page.tsx

```
@@ -0,0 +1,259 @@
+			'@type': 'Person',
+			name: config.author,
+		},
+		description: description,
+		...(imageUrl && { thumbnailUrl: imageUrl }),
+		url: `${env.NEXT_PUBLIC_URL}/workshops/${slug}`,
+	}
+
+	return (
+		<script
+			type="application/ld+json"
+			dangerouslySetInnerHTML={{ __html: JSON.stringify(jsonLd) }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)